app/konnect/servicehub/plugins.md

```diff
@@ -50,20 +51,19 @@ supports.
 
 ## Custom plugins
 
-Currently, there is no way to add a custom plugin
-directly through the {{site.konnect_saas}} application. Contact Kong
-Support to get them manually added to your organization.
+{{site.konnect_saas}} supports the use of custom plugins. You can write new custom plugins using this [template](https://github.com/Kong/kong-plugin) as a guide. Every custom plugin must meet the following requirements: 
 
-Custom plugins must not have the following:
-
-* Admin API extensions: No `api.lua` file
-* Custom plugin database tables: No `dao.lua` file
-* Custom function validations: No function definitions in `schema.lua`
-* Code that runs on the control plane in the plugin handler:
-  * No `init_worker` callback
-  * No Lua code outside of the top-level functions
-* Third-party library dependencies: No `require()` calls to modules that are
-not bundled by default with {{site.konnect_product_name}}
+* Admin API extensions must not contain an `api.lua` file. 
+* Custom plugin database tables must not contain a `dao.lua` file.
+* Custom function validations:
```

Comment:
For this bullet and the other bullet with subbullets ("Third-party library dependencies:"), I might combine this into one bullet point like "Custom function validations must be written in Lua and self-contain within the schema." But I'm not sure if you nest bullet points in other parts of the docs.

Reply:
Great point.